Add Cloud Topics to Enterprise Feature License table (#1708)
redpanda-data/documentation: modules/get-started/pages/licensing/overview.adoc

| Records detailed logs of cluster activities for compliance and monitoring.
| Read access to the audit log topic is denied, but logging continues.

> | xref:develop/manage-topics/cloud-topics.adoc[Cloud Topics]

Author: ```suggestion
| xref:develop:manage-topics/cloud-topics.adoc[Cloud Topics]
```

| A Redpanda topic type that uses durable object storage as the primary backing store instead of local disk replication.

> | Cloud Topics cannot be created. Existing Cloud Topics cannot be modified. Additional partitions cannot be added to existing Cloud Topics. Major upgrades are blocked when in a violation state.

Author: ```suggestion
| New Cloud Topics cannot be created. Existing Cloud Topics cannot be modified (including the addition or modification of partitions for existing Cloud Topics). Major upgrades are blocked when in a violation state.
```

> | New Cloud Topics cannot be created. Existing Cloud Topics cannot be modified (including the addition or modification of partitions for existing Cloud Topics). Major upgrades are blocked when in a violation state.

Author: ```suggestion
| New Cloud Topics cannot be created. Existing Cloud Topics cannot be modified, including adding or modifying partitions. Major upgrades are blocked when in a violation state.
```
